### ISSUE TITLE
Docker image not working

### ISSUE THREAD

**Issue by victorgomezg93** [Qovery/Replibyte#147]
Hi, I'm trying to run the docker image to test just following the guide:

```
git clone https://github.com/Qovery/replibyte.git

# Build image with Docker
docker build -t replibyte -f Dockerfile .

# Run RepliByte
docker run -v $(pwd)/examples:/examples/ replibyte -c /examples/replibyte.yaml transformer list
```

I'm doing it in Ubuntu 20.04 and when I try to make the docker run I have this error:
```
No such file or directory (os error 2)
```

Maybe some package is missing? If I make the installation from source or from linux the same message appear.

Thank you very much for your help

**Comment by fabriceclementz** [comment]
Hi @victorgomezg93 
I think you should use the official image here so you don't need to build it locally (available since https://github.com/Qovery/Replibyte/pull/128), 

Here is a working example assuming you have a config file named `replibyte.yaml`, this example show you how to mount a volume `datastore` via ` -v "$(pwd)/datastore:/datastore" \`, this line is optional if you're using S3 for example.

Mounting your configuration file at `/replibyte.yaml:ro` is required.

```sh
docker run -it --rm --name replibyte \
  -v "$(pwd)/replibyte.yaml:/replibyte.yaml:ro" \
  -v "$(pwd)/datastore:/datastore" \
  ghcr.io/qovery/replibyte --config replibyte.yaml transformer list
```

I will update the documentation here https://www.replibyte.com/docs/getting-started/installation

Hope this help you.

**Comment by victorgomezg93** [comment]
@fabriceclementz  I think I have the same problem with the official image, I've used docker run with the datastore volume and without but if you see there is a "No such file or directory (os error 2)" in the last line like with the other image. 
![image](https://user-images.githubusercontent.com/45047366/172326078-29f103ef-4f86-4106-8f32-b246bf45abb7.png)

**Comment by fabriceclementz** [comment]
I will try to reproduce asap @victorgomezg93

**Comment by victorgomezg93** [comment]
Hi @fabriceclementz I'm trying to make it run from local in the ubuntu to check issues. Can maybe be related with the uri? I don't have user and password created in the mongo database maybe that's why is failing.

I have been using in source and destination
```
connection_uri: mongodb://172.17.0.6:27017/test
````

and the error message is 
```
error parsing command line options: illegal argument combination: cannot specify --authenticationDatabase and --uri
2022-06-08T15:32:20.387+0200	try 'mongorestore --help' for more information
```

So I created a new user and password to check if maybe the problem was the absence of this but is not working also, same message with this:

```
mongodb://myUserAdmin:abc123@172.17.0.6:27017/test

```

Saw this https://github.com/Qovery/Replibyte/pull/38

And I tried
```
mongodb://myUserAdmin:abc123@172.17.0.6:27017/corporate_staging?authSource=admin
```
But still same message, I can't use both options I think https://github.com/Qovery/Replibyte/blob/main/replibyte/src/destination/mongodb.rs#L42
